projectEuler/problem1.js

```diff
@@ -1,4 +1,5 @@
 // prompt: find the sum of all the multiples of 3 and 5 below 100
+
 function multSum(valOne, valTwo, max) {
   var sumOne = 0;
   var sumTwo = 0;
```

Comment:
It would be worth tidying up this function in problem 1 to use `let` instead of `var`